### PR TITLE
Enable notification of past events

### PR DESCRIPTION
**Pull request by jbenito3**
- Added 'timedelta' to the configuration file so we can specify
  the timespan (e.g. 30m). If negative (e.g. -3h), events from the past
  will be fetched and notified.

### CODE REVIEW

File: indico_mattermost_bot.py [hunk 1]
```diff
@@ -24,9 +50,16 @@ def _split(text):
 
 
 def _process_bots(config):
-    channel_ids = [section for section in config.sections() if section.startswith('channel_')]
-    bot_ids = [section for section in config.sections() if section.startswith('bot_')]
-    channel_hooks = {cid[8:]: {'hook_url': config[cid]['hook_url'], 'text': config[cid]['text']} for cid in channel_ids}
+    channel_ids = [section for section
+                   in config.sections() if section.startswith('channel_')]
```
Comment (pferreir) [inline indico_mattermost_bot.py]: 80 chars? 120 please 😉

File: indico_mattermost_bot.py [hunk 2]
```diff
@@ -61,7 +95,10 @@ def notify(event, bot, channels):
             'username': bot['nickname'],
             'icon_url': bot['image_url']
         }
-        requests.post(url, data={b'payload': json.dumps(payload).encode('utf-8')})
+        requests.post(
+            url,
+            data={b'payload': json.dumps(payload).encode('utf-8')}
+        )
```
Comment (pferreir) [inline indico_mattermost_bot.py]: I guess same here.

File: indico_mattermost_bot.py [hunk 3]
```diff
@@ -78,6 +115,8 @@ def read_config(config_file):
         'channels': channels
     }
 
+def is_fetching_past_events(bot):
```
Comment (pferreir) [inline indico_mattermost_bot.py]: I'd underscore the function name, since we're doing the same above (e.g. `_dt(...)`).

**Comment by pferreir** [comment]
Seems fine, apart from the 80 char limit 😉

**Comment by jbenito3** [comment]
@pferreir changes done! 🐑  it! 😸

**Comment by pferreir** [comment]
LGTM! 👍

**Pull request merged**